spec/fixtures/vcr_cassettes/Autocompleting_WorkVersion_metadata_with_data_from_RMD/Submitting_the_Autcomplete_For_Open_Access_Works_form/when_work_type_is_not_a_scholarly_work/does_not_render_Autocomplete_For_Open_Access_Works_form.yml

```diff
@@ -0,0 +1,46 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://sessions.bugsnag.com/
+    body:
+      encoding: UTF-8
+      string: '{"notifier":{"name":"Ruby Bugsnag Notifier","url":"https://www.bugsnag.com","version":"6.27.1"},"device":{"hostname":"L5-LST-JML8735","runtimeVersions":{"ruby":"3.1.2","rails":"6.1.7.8","rack":"2.2.9"}},"app":{"version":null,"releaseStage":"development","type":"active
+        job"},"sessionCounts":[{"startedAt":"2024-10-24T20:29:00","sessionsStarted":1},{"startedAt":"2024-10-24T20:30:00","sessionsStarted":1}]}'
+    headers:
+      Bugsnag-Api-Key:
+      - 4a5bd13239520eda7e393cf0c8cb855a
```

Comment:
I'm curious why bugsnag is suddenly being triggered?  There must be some error, but it doesn't seem to be sending anything.  We should probably add bugsnag to the list of urls that webmock ignores.  Also, there's an API key in here, so we'll need to remove these.